alert-common/src/test/java/com/synopsys/integration/alert/common/channel/message/MessageSplitterTest.java

```diff
@@ -0,0 +1,32 @@
+package com.synopsys.integration.alert.common.channel.message;
+
+import java.util.List;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+class MessageSplitterTest {
+
+    @Test
+    void messageSplitterTest() {
+        int limit = 10;
+        String lineSeparator = "\n";
+        char delimiter = ' ';
+        MessageSplitter messageSplitter = new MessageSplitter(limit, lineSeparator, delimiter);
+        List<String> splitMessages = messageSplitter.splitMessages(List.of(
+            "category (ADD) - component-value \n",
+            // Can produce a StackOverflowError if the split index is 0
+            " testing_of_splitting_index_0 ",
+            "",
+            " "
+        ));
+
+        for (String splitMessage : splitMessages) {
+            System.out.println("'" + splitMessage + "'");
+            Assertions.assertTrue(splitMessage.length() <= limit,
```

Comment:
I believe we usually will import the assertions we need like assertTrue and not the entire Assertions package